contrib/python/podman/podman/libs/containers.py

```diff
@@ -237,9 +237,9 @@ def delete_stopped(self):
             results = podman.DeleteStoppedContainers()
         return results['containers']
 
-    def get(self, id):
+    def get(self, id_):
```

Comment:
Is there a reason this is id_ instead of _id?

Reply:
According to PEP8:

_id is for "private" variables while id_ is for not overriding a Python keyword.